python/lsst/analysis/ap/plotImageSubtractionCutouts.py

```diff
@@ -388,21 +396,34 @@ def generate_image(self, science, template, difference, center, scale,
         """
         if (source is None) ^ (flags is None):
             raise RuntimeError("Must pass both `source` and `flags` together.")
+        numpy_cutouts = {}
         if not self.config.use_footprint:
```

Comment:
As we discussed I think it is correct to only save numpy cutouts with the same cutout size (use footprint False).  However, you should raise an error if `use_footprint` and `save_as_numpy` are both true.

Reply:
Added code for raising a Runtime Error if both `use_footprint` and `save_as_numpy` are true.